pkg/fuzzy/fuzzy_test.go

```diff
@@ -19,8 +20,7 @@ import (
 // ---–––--------------Actual Tests------------------–––---
 
 func Test_TranslateMock(t *testing.T) {
```

Comment:
Instead of random messages, I suggest concrete messages, with all mf2 types, Simple,Complex, Matcher.
Here are my tests, which should cover all (positive) cases.
With the Translate() functionality behaving as I expect it to behave
```go
func Test_TranslateMock(t *testing.T) {
	// NOTE: AWS skipped for now until fuzzy translation is fixed.
	t.Parallel()

	ctx := context.Background()

	allMocks(t, func(t *testing.T, mock Translator) {
		tests := []struct {
			translation *model.Translation
			targetLang  language.Tag
			name        string
		}{
			{
				name:       "Simple message",
				targetLang: language.Latvian,
				translation: &model.Translation{
					Language: language.English,
					Original: true,
					Messages: []model.Message{
						{
							ID:      "1",
							Message: "Hello World!",
							Status:  model.MessageStatusTranslated,
						},
					},
				},
			},
			{
				name:       "Complex message quoted pattern",
				targetLang: language.Latvian,
				translation: &model.Translation{
					Language: language.English,
					Original: false,
					Messages: []model.Message{
						{
							ID: "1",
							Message: `.local $format = { python-format }
.local $name = { |{name}| }
{{Hello, { $name }!}}`,
							Status: model.MessageStatusUntranslated,
						},
					},
				},
			},
			{
				name:       "Complex message matcher",
				targetLang: language.Latvian,
				translation: &model.Translation{
					Language: language.Russian,
					Original: true,
					Messages: []model.Message{
						{
							ID: "1",
							Message: `.local $format = { python-format }
.local $suggestion = { |%(suggestion)s| }
.local $undefinedParameter = { |%(undefinedParameter)s| }
.local $firstSuggestions = { |%(firstSuggestions)s| }
.local $lastSuggestion = { |%(lastSuggestion)s| }
.match { $count }
1 {{{ $suggestion } вместо \\"{ $undefinedParameter }\\"?}}
2 {{
{ $firstSuggestions } или { $lastSuggestion } вместо 
\\"{ $undefinedParameter }\\"?}}
* {{
{ $firstSuggestions } или { $lastSuggestion } вместо 
\\"{ $undefinedParameter }\\"?}}`,
							Status: model.MessageStatusTranslated,
						},
					},
				},
			},
			// ID1 remains as is, ID2 is sent to translation.
			{
				name:       "One Translated message one not",
				targetLang: language.Latvian,
				translation: &model.Translation{
					Language: language.English,
					Original: false,
					Messages: []model.Message{
						{
							ID:      "1",
							Message: "Hello World!",
							Status:  model.MessageStatusTranslated,
						},
						{
							ID:      "2",
							Message: "Hello World!",
							Status:  model.MessageStatusUntranslated,
						},
					},
				},
			},
		}

		for _, tt := range tests {
			tt := tt
			t.Run(tt.name, func(t *testing.T) {
				t.Parallel()

				translated, err := mock.Translate(ctx, tt.translation, language.Latvian)
				require.NoError(t, err)

				// Check the that the translated translation have the correct language.
				require.Equal(t, tt.targetLang, translated.Language, "want: %s, got: %s", tt.translation.Language, translated.Language) //nolint:lll

				// Check that it is not original
				require.False(t, translated.Original, "translation cannot be original")

				// Check that length matches.
				require.Len(t, translated.Messages, len(tt.translation.Messages), "message count mismatch")

				for i, actualMsg := range translated.Messages {
					require.NotEmpty(t, actualMsg.Message)
					// In mock, we return the same message, so it should be the same.
					require.Equal(t, tt.translation.Messages[i].Message, actualMsg.Message)

					expectedStatus := tt.translation.Messages[i].Status // default: fuzzy or translated
					// If the message is not original and it is not translated, then it should be fuzzy.
					if !tt.translation.Original && tt.translation.Messages[i].Status != model.MessageStatusTranslated {
						expectedStatus = model.MessageStatusFuzzy
					}

					require.Equal(t, expectedStatus, actualMsg.Status, "want: %s, got: %s", expectedStatus, actualMsg.Status) //nolint:lll
				}
			})
		}
	})
}
```

Reply:
random already creates 2 types - simple, complex and complex with matcher.